doc/source/download.rst

```diff
@@ -10,6 +10,160 @@ Download
+available at https://anaconda.org/conda-forge/gdal.
+
+.. only:: html
+
+    Latest version: |Conda badge|
+
+    .. |Conda badge| image:: https://anaconda.org/conda-forge/gdal/badges/version.svg
+        :target: https://anaconda.org/conda-forge/gdal
+
+::
+
+    conda install [-c channel] [package...]
+
+
+::
+
+    conda install -c conda-forge gdal
+
+
+To install the Arrow and Parquet drivers as plugins:
+
+::
+
+    conda install -c conda-forge libgdal-arrow-parquet
+
+
+GDAL master Conda builds
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+GDAL master builds are available in the `gdal-master <https://anaconda.org/gdal-master/gdal>`__
+channel. They are based on dependencies from the ``conda-forge`` channel.
+
+First, install mamba into the ``base`` environment, create a dedicated ``gdal_master_env``
+environment, and then activate the dedicated ``gdal_master_env`` environment.
+
+::
+
+    conda update -n base -c conda-forge conda
+    conda install -n base --override-channels -c conda-forge mamba 'python_abi=*=*cp*'
+    conda create --name gdal_master_env
+    conda activate gdal_master_env
+
+Then install GDAL from the ``gdal-master`` channel:
+
+::
+
+    mamba install -c gdal-master gdal
+    mamba install -c gdal-master libgdal-arrow-parquet # if you need the Arrow and Parquet drivers
+
+
+Vcpkg
+................................................................................
+
+The gdal port in vcpkg is kept up to date by Microsoft team members and community contributors.
+The url of vcpkg is: https://github.com/Microsoft/vcpkg .
+You can download and install gdal using the vcpkg dependency manager:
+
+::
+
+    git clone https://github.com/Microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh  # ./bootstrap-vcpkg.bat for Windows
+    ./vcpkg integrate install
+    ./vcpkg install gdal
+
+If the version is out of date, please `create an issue or pull request <https://github.com/Microsoft/vcpkg>`__ on the vcpkg repository.
+
+Spack
+................................................................................
+
+Spack is a package management tool designed to support multiple versions and
+configurations of software on a wide variety of platforms and environments.
+It was designed for large supercomputing centers. Spack builds packages from
+sources, and allows tweaking their configurations.
+
+You can find information about GDAL in Spack at
+https://spack.readthedocs.io/en/latest/package_list.html#gdal
+
+For the default GDAL build with a reduced number of drivers:
+
+::
+
+    git clone -c feature.manyFiles=true https://github.com/spack/spack.git
+    cd spack/bin
+    ./spack install gdal
+
+For a build with netcdf driver enabled:
+
+::
+
+    ./spack install gdal +netcdf
+
+
+Linux Docker images
+................................................................................
+
+Images with nightly builds of GDAL master and tagged releases are available at
+`Docker Hub <https://hub.docker.com/r/osgeo/gdal/tags>`_
```

Comment:
hum, that doesn't match the current version of that page which rightly points  to the github container registry (we've abandonned Docker Hub. actually that's more the reverse...). Could you check or redo your cut & move changes, because it appears you used not up-to-date material ?